storyscript/compiler/semantics/ExpressionResolver.py

```diff
@@ -154,6 +154,32 @@ def nary_expression(self, tree, op, values):
             self.implicit_cast(tree, val, values)
         return base_symbol(val)
 
+    @staticmethod
+    def type_cast_entity(expr_node, target_type):
+        """
+        Type casts the given expr_node with an entity to the target_type.
+        This is done by creating a new AST for the expr_node with an
+        as_operator
+        Note: This does not perform any checks around feasibility of performing
+        the type cast operation and depends upon the caller to have had
+        performed these checks before making the call.
+        """
+        assert expr_node.data == 'expression'
+        casted_type = SymbolExpressionVisitor.type_to_tree(
```

Comment:
```suggestion
        cast_type = SymbolExpressionVisitor.type_to_tree(
```

Let's improve this slightly if we're already at it.